test/algconvergence/partitioned_methods_tests.jl

```diff
@@ -368,6 +378,76 @@ sol = solve(prob, ERKN5(), reltol = 1e-8)
 sol = solve(prob, ERKN7(), reltol = 1e-8)
 @test length(sol.u) < 38
 
+# Testing generalized Runge-Kutte-Nyström methods on velocity dependend ODEs with the damped oscillator
+println("In Place")
+
+# Damped oscillator
+prob = ODEProblem(DynamicalODEFunction{false}((du, u, p, t) -> -u - 0.5 * du,
+        (du, u, p, t) -> du,
+        analytic = (du0_u0, p, t) -> OrdinaryDiffEq.SciMLBase.ArrayPartition([
+                exp(-t / 4) / 15 * (15 * du0_u0[1] * cos(sqrt(15) * t / 4) -
+                 sqrt(15) * (du0_u0[1] + 4 * du0_u0[2]) * sin(sqrt(15) * t / 4)),
+            ], # du
+            [
+                exp(-t / 4) / 15 * (15 * du0_u0[2] * cos(sqrt(15) * t / 4) +
+                 sqrt(15) * (4 * du0_u0[1] + du0_u0[2]) * sin(sqrt(15) * t / 4)),
+            ])),
+    OrdinaryDiffEq.SciMLBase.ArrayPartition([0.0], [1.0]), # du0, u0
+    (0.0, 10.0), # tspan
+    DiffEqBase.NullParameters(), # p
+    SecondOrderODEProblem{false}())
+
+dts = 1.0 ./ 2.0 .^ (5:-1:0)
+sim = test_convergence(dts, prob, Nystrom4(), dense_errors = true)
+@test sim.𝒪est[:l2]≈4 rtol=1e-1
+@test sim.𝒪est[:L2]≈4 rtol=1e-1
+sim = test_convergence(dts, prob, FineRKN4(), dense_errors = true)
+@test sim.𝒪est[:l2]≈4 rtol=1e-1
+@test sim.𝒪est[:L2]≈4 rtol=1e-1
+sim = test_convergence(dts, prob, FineRKN5(), dense_errors = true)
+@test sim.𝒪est[:l2]≈5 rtol=1e-1
+@test sim.𝒪est[:L2]≈4 rtol=1e-1
```

Comment:
Interesting! So the methods seem to have some superconvergence properties for the harmonic oscillator without damping but not for the version with damping used here.
(nothing blocking this PR from being merged, just an observation)